src/notation/qml/MuseScore/NotationScene/internal/HarpPedalGridColumn.qml

```diff
@@ -0,0 +1,105 @@
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+ColumnLayout {
+    property var repeaterModel
+    property QtObject model: harpModel
+
+    property var buttonId: repeaterModel.buttonsId
+
+    property NavigationPanel navPanel: null
+
+    Layout.preferredWidth: 20
+    Layout.preferredHeight: 120
+
+    function checkPedalState(string, state) {
+        var pedalState = harpModel.pedalState
+        if (pedalState[string] == state) {
+            return true
+        }
+        return false
+    }
+
+    function updatePedalState(string, state) {
+        root.pedalState[string] = state
+        harpModel.setPedalState(root.pedalState)
+    }
+
+    StyledTextLabel {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+
+        font: ui.theme.largeBodyBoldFont
+
+        text: repeaterModel.stringName
+    }
+
+    RoundedRadioButton {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        spacing: 5
+        checked: checkPedalState(repeaterModel.stringId, 0)
+
+        navigation.name: repeaterModel.stringName + " Flat"
+        navigation.panel: navPanel
+        navigation.order: repeaterModel.stringId * 3 + 1
+        navigation.accessible.name: repeaterModel.stringName + qsTrc("notation", " flat")
+
+        onToggled: updatePedalState(repeaterModel.stringId, 0)
+    }
+
+    RoundedRadioButton {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        spacing: 5
+        checked: checkPedalState(repeaterModel.stringId, 1)
+
+        navigation.name: repeaterModel.stringName + " Natural"
+        navigation.panel: navPanel
+        navigation.order: repeaterModel.stringId * 3 + 2
+        navigation.accessible.name: repeaterModel.stringName + qsTrc("notation", " natural")
+
+        onToggled: updatePedalState(repeaterModel.stringId, 1)
+    }
+
+    RoundedRadioButton {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        spacing: 5
+        checked: checkPedalState(repeaterModel.stringId, 2)
+
+        navigation.name: repeaterModel.stringName + " Sharp"
+        navigation.panel: navPanel
+        navigation.order: repeaterModel.stringId * 3 + 3
+        navigation.accessible.name: repeaterModel.stringName + qsTrc("notation", " sharp")
```

Comment:
We got to be careful here regarding translations. IIRC, in German there is no word for "sharp" or "flat" but they include the accidental in the note name (e.g. F Sharp is Fis). (@Jojo-Schmitz Please correct me if I'm wrong 😉)

So I think we'll need a matrix of 3x7 translatable strings, namely the cartesian product of [flat, natural, sharp] and the 7 notes.

Reply:
Well, we'd have "Auflösungszeichen", "Kreuz" and "Be", but not as a postfix to a notename or whatever `stringName` may contain? Also a leading space in a translatable string is a bad idea.